bindings/protocols/bacnet/index.html

```diff
@@ -75,13 +75,13 @@
     <section id='abstract'>
         <p>
             In the context of the Web of Things (WoT), a Binding Template is a blueprint that gives guidance on how to
-            implement a specific IoT protocol, data format or IoT platform.
+            implement a specific IoT protocol, data format, or IoT platform.
             The Core Binding Templates specification explains the overall mechanism and requirements for any binding to follow.
             This document gives implementation guidelines regarding BACnet [[BACnet]], a building automation and control networking protocol.
         </p>
         <p>
             More specifically, this document defines a set of vocabulary terms that can be used inside a Thing Description document,
-            and associated rules which allow to describe WoT operations using BACnet over the network.
+            and associated rules that allow to description of WoT operations using BACnet over the network.
```

Comment:
```suggestion
            and associated rules that allow to describe WoT operations using BACnet over the network.
```

The proposed text does not sound right